cmd/metal-api/internal/service/partition-service.go

```diff
@@ -103,11 +103,23 @@ func (r partitionResource) webService() *restful.WebService {
 		Returns(http.StatusConflict, "Conflict", httperrors.HTTPErrorResponse{}).
 		DefaultReturns("Error", httperrors.HTTPErrorResponse{}))
 
+	// Deprecated, can be removed in the future
```

Comment:
as this is only used by "us" im fine removing it immediately.

Or is there a place in the cloud-api where we check for available machines ?

Reply:
Main reason for this decision to me was to have old metalctl versions still working for a certain time. In cloud-api we also call PartitionCapacity.